tests/tests.py

```diff
@@ -1038,66 +1044,42 @@ def test_plantcv_transform_warp_smaller():
     bimg_small = cv2.resize(bimg, (200,300)) #not sure why INTER_NEAREST doesn't preserve values
     bimg_small[bimg_small>0]=255
     mrow, mcol = bimg_small.shape
-    vrow, vcol, vdepth = img.shape
-    pcv.params.debug = None
-    mask_warped = pcv.transform.warp(bimg_small, img[:,:,2],
-                                    pts = [(0,0),(mcol-1,0),(mcol-1,mrow-1),(0,mrow-1)],
-                                    refpts = [(0,0),(vcol-1,0),(vcol-1,vrow-1),(0,vrow-1)])
-    pcv.params.debug = 'plot'
-    mask_warped_plot = pcv.transform.warp(bimg_small, img[:,:,2],
-                                pts = [(0,0),(mcol-1,0),(mcol-1,mrow-1),(0,mrow-1)],
-                                refpts = [(0,0),(vcol-1,0),(vcol-1,vrow-1),(0,vrow-1)])
-
-    assert np.count_nonzero(mask_warped)==93142
-    assert np.count_nonzero(mask_warped_plot)==93142
 
-
-def test_plantcv_transform_warp_larger():
-    img = cv2.imread(os.path.join(TEST_DATA, TEST_INPUT_COLOR),-1)
-    gimg = cv2.imread(os.path.join(TEST_DATA, TEST_INPUT_GRAY),-1)
-    gimg_large = cv2.resize(gimg, (5000,7000))
-    mrow, mcol = gimg_large.shape
     vrow, vcol, vdepth = img.shape
-    pcv.params.debug='print'
-    mask_warped_print = pcv.transform.warp(gimg_large, img,
-                                    pts = [(0,0),(mcol-1,0),(mcol-1,mrow-1),(0,mrow-1)],
-                                    refpts = [(0,0),(vcol-1,0),(vcol-1,vrow-1),(0,vrow-1)])
-
-    assert np.sum(mask_warped_print)==83103814
+    pcv.params.debug = None
+    warped_img,  mat = pcv.transform.warp(img, img, pts = [(0,0),(vcol-1,0),(vcol-1,vrow-1),(0,vrow-1)], refpts = [(0,0),(vcol-1,0),(vcol-1,vrow-1),(0,vrow-1)])
+    assert mat.shape == (3, 3) and warped_img.dtype == img.dtype
 
+    _ = pcv.transform.warp_align(img, mat)
+    warped_img_ = pcv.transform.warp_align(img, mat, img)
+    assert warped_img_.dtype == img.dtype
```

Comment:
based on what @nfahlgren  said yesterday we can't have multiple assert per test because assert acts like return.  also look like this checks the dtype same as before

Reply:
I've parameterized the tests for `transform.warp` and `transform.warp_align`